styleguide/components/31-molecules/nav-filter/nav-filter.config.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+  name: 'Nav filter',
+  handle: 'nav-filter',
+  status: 'alpha',
+  preview: '@preview',
+  context: {
+    list: [
+      '<a href="/ga-voor-een-onvergetelijke-schoolstage" class="active">Alle opleidingsniveaus</a>',
```

Comment:
English please
Also, if it's TOC the links should be anchor links so screenreaders say "same page link".